### PR TITLE
feat: add tailwind css for styling

### PR DESCRIPTION
**Pull request by lidusol**
# Description
- Used [TailwindCSS](https://tailwindcss.com/) to customize components by adding [daisyUI](https://daisyui.com/)

## Type of Change

- [x] feat (A new feature)

## Reference:

Home Page:
screencapture-localhost-8080-2024-06-25-11_32_44

Create Movie:
screencapture-localhost-8080-2024-06-23-00_59_07

Edit Movie:
Screenshot 2024-06-25 at 11 43 37 in the morning

Movie Detail:
screencapture-localhost-8080-2024-06-23-00_59_34

### CODE REVIEW

**Comment by NeoMax97** [comment]
Hey @lidusol, great work so far! I found some stuff while reviewing the changes. Please look into them.

1. Console.logs are still present (Can be removed at the end as a cleanup task)
2. I see the following error on my console. Something about mutating props.
image

3. When I create a movie, I don't see a field for adding an image, so new movies no poster.
4. New data is only fetched on refresh (You raised this on Slack already)
5. I see we have an EditMovie component, but there's no way for me to access it from UI. After adding it to the routes and accessing it manually, the page seems almost blank. We have an Update button, but clicking it just throws an error. No fields are visible. Maybe it's only an error when accessing manually, but works fine when navigating from within the App?

CC: @Hardik-K-Patel, please have a look too, in case you're aware of how to fix some of these issues already.

**Comment by Hardik-K-Patel** [comment]
1. Console.logs are still present (Can be removed at the end as a cleanup task) = As @lidusol said We can keep it for debugging purposes.

2. I see the following error on my console. Something about mutating props. = The warning might be caused by the version we are currently using.
3. When I create a movie, I don't see a field for adding an image, so new movies no poster. = 
Yes, I have added the title and body for now to keep it simple, as we are currently in the learning phase.
4. New data is only fetched on refresh (You raised this on Slack already) = 
I think it's the default behavior of Vue.js.
5. I see we have an EditMovie component, but there's no way for me to access it from UI. After adding it to the routes and accessing it manually, the page seems almost blank. We have an Update button, but clicking it just throws an error. No fields are visible. Maybe it's only an error when accessing manually, but works fine when navigating from within the App? = @lidusol can we check missing edit button on the movie listing page.

@NeoMax97 regarding points 1, 2, 3, and 4, since we are currently in the learning phase, we can focus on improving these areas in the future as we gain more experience with Vue.js. For now, basic integration is sufficient for the demo.

File: src/components/HomePage.vue [hunk 1]
```diff
@@ -1,96 +1,240 @@
...
+          </div>
+        </div>
+      </div>
 
-                    <div class="col-md-3">
-                        <router-link :to="{ name: 'delete', params: { delete: movie.nid[0].value } }">Delete Movie</router-link>
-                    </div>
-                    <div class="col-md-3">
-                        <router-link :to="{ name: 'edit', params: { edit: movie.nid[0].value } }">Edit Movie</router-link>
-                    </div>
```
Comment (Hardik-K-Patel) [inline src/components/HomePage.vue:50]: @lidusol , this was the edit link.

Comment (lidusol) [reply src/components/HomePage.vue:50]: @Hardik-K-Patel I have included it now

**Pull request merged**